doc/LIMITS.md

```diff
@@ -27,34 +27,53 @@ const node = await createLibp2pNode({
   connectionManager: {
     /**
      * The total number of connections allowed to be open at one time
+     *
+     * This field is optional, the default value is shown
      */
     maxConnections: 200,
 
     /**
      * If the number of open connections goes below this number, the node
-     * will try to connect to nearby peers from the peer store
+     * will try to connect to randomly selected peers from the peer store
+     *
+     * This field is optional, the default value is shown
      */
     minConnections: 20,
 
     /**
      * How many connections can be open but not yet upgraded
+     *
+     * This field is optional, the default value is shown
      */
     maxIncomingPendingConnections: 10
   }
 })
 ```
 
```

Comment:
```suggestion
```

The latest default values can be found [here](https://github.com/libp2p/js-libp2p/blob/master/src/connection-manager/index.ts)
```

I'm not sure if this is true or not, but basic idea is to link to the source.